internal/etw/eventopt.go

```diff
@@ -36,6 +36,12 @@ func WithKeyword(keyword uint64) EventOpt {
 	}
 }
 
+func WithChannel(channel Channel) EventOpt {
+	return func(options *eventOptions) {
+		options.descriptor.Channel = channel

```

Comment:
Where do you even use this?

Reply:
We don't use it anywhere. I added it when testing changing channel, and decided to leave it in rather than delete the code once it was written.